src/main/java/jep/PyConfig.java

```diff
@@ -67,9 +252,11 @@ public class PyConfig {
      * @param noSiteFlag
      *            value to pass to Python for Py_NoSiteFlag
      * @return a reference to this PyConfig
+     * @deprecated Use {@link #setSiteImport(int)} instead.
      */
+    @Deprecated
     public PyConfig setNoSiteFlag(int noSiteFlag) {
-        this.noSiteFlag = noSiteFlag;
+        this.siteImport = (noSiteFlag == 0) ? 1 : 0;
```

Comment:
Mostly stylistic choice, but I think it might be slightly cleaner if you had these deprecated methods call the new non-deprecated methods to set the values.  Just a thought.

Reply:
I agree that is nicer, fixed.